README.md

```diff
@@ -298,6 +311,83 @@ publish.standard();
                 "dev"
             </td>
         </tr>
+        <tr>
+            <td>
+                <code>remoteName</code>
+            </td>
+            <td>
+                The remote repository to push version control tag to.
+            </td>
+            <td>
+                "upstream"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>changesHint</code>
+            </td>
+            <td>
+                A hint for addressing uncommitted changes.
+            </td>
+            <td>
+                "Address uncommitted changes: Commit \"git commit -a\", Stash \"git stash\" or Clean \"git reset --hard\"\n"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>checkRemoteHint</code>
+            </td>
+            <td>
+                A hint for addressing an issue where the remote repository cannot be found.
+            </td>
+            <td>
+                "Run \"git remote -v\" for a list of available remote repositories.\n"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>publishHint</code>
+            </td>
+            <td>
+                A hint for addressing an issue where publishing to the registry fails.
+            </td>
+            <td>
+                "Ensure that you have access to publish to the registry and that the current version does not already exist.\n"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>publishDevHint</code>
+            </td>
+            <td>
+                A hint for addressing an issue where publishing to the registry fails.
+            </td>
+            <td>
+                "Ensure that you have access to publish to the registry and that the current version does not already exist.\nIf the tag already exists use a new tag name or run \"npm dist-tag rm ${packageName} ${tag}\" to remove the existing one.\nAlso ensure that the tag name is valid (i.e. doesn't conform to a valid semver range like v1.4 or 1.4).\n"
```

Comment:
The last sentence is puzzlingly worded - is it the case that a tag name CANNOT correspond to a semver version, or is it the opposite?